corehq/apps/data_vault/models.py

```diff
@@ -0,0 +1,15 @@
+import secrets
+
+from django.db import models
+
+
+class VaultStore(models.Model):
+    id = models.BigAutoField(primary_key=True)
+    key = models.CharField(max_length=25, unique=True)
+    value = models.CharField(max_length=255)
+    identifier = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
this doesn't seem to have any value since it will only be one value. It can be added later if we need to store different kinds of data here

Reply:
I am just waiting on one final confirmation from app team, aadhaar is stored in different ways, as a text, as a part of the raw_aadhaar_string or some other property, just kept this column in advance if there are more places to be covered.